bootstrap/main.py

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env python3
+
+import json
+import os
+import pathlib
+import shutil
+import sys
+import time
+from typing import Any
+
+import docker
+
+client = docker.from_env()
+
+default_file_path = pathlib.Path("/startup.json.default")
+config_file_path = pathlib.Path("/config/startup.json")
+
+CORE_CONTAINER_NAME = "companion_core"
+
+
+def ensure_dir(file_path: str) -> None:
+    """Makes sure that the file path exists
+
+    Args:
+        file_path (str): path to check/make
+    """
+    directory = os.path.dirname(file_path)
+    if not os.path.exists(directory):
+        os.makedirs(directory)
+
+
+def stop_old_container() -> None:
+    """ Looks for an old core container and stops it if found"""
+    try:
+        old_container = client.containers.get(CORE_CONTAINER_NAME)
+        old_container.stop()
+        old_container.remove()
+    except Exception:
+        pass
```

Comment:
you should at least warn that something is wrong, we don't fix our problem keeping it in silence.
`warnings.warn("Oh jezz, something is wrong rick: {exception}")`